README.md

```diff
@@ -87,14 +87,7 @@ additional metrics!
 >	* kube_node_status_capacity_nvidia_gpu_cards
 >	* kube_node_status_allocatable_nvidia_gpu_cards
 >
->	are in alpha stage and will be deprecated when the kubernetes gpu support is final in 1.9 version.
->
-> **collectors**
-> * v2alpha1/cronjob
->
-> If users want to enable this feature when kubernetes version larger than 1.7,
-> It must be configured, with the following parameter setting for apiserver.
-> `--runtime-config=batch/v2alpha1=true`
+>	are in alpha stage and will be deprecated when the kubernetes accelerator feature support is deprecated in version v1.11.
```

Comment:
> kubernetes accelerator feature support is deprecated in version v1.11.

Kubernetes accelerator support is already deprecated in 1.10. The PR to completely remove it is merged in master, so it will be completely removed in 1.11.

Reply:
@mindprince Thanks for the more accurate deprecation procudure about Kubernetes accelerator. Will fire another PR to update this info.

Reply:
@mindprince FRI #417